core/build.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
     implementation 'commons-io:commons-io:2.8.0'
     implementation 'com.univocity:univocity-parsers:2.9.0'
     implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.google.geometry:s2-geometry:2.0.0'
```

Comment:
🎉

Reply:
Fresh, tiny and neat library just for our favorite GTFS validator!